testframes/test_forward_solution/test_forward_solution.cpp

```diff
@@ -130,7 +130,7 @@ void TestForwardSolution::computeForward()
     settings.transname.clear();
     settings.bemname = QDir::currentPath()+QCoreApplication::applicationDirPath() + "/MNE-sample-data/subjects/sample/bem/sample-5120-5120-5120-bem.fif";
```

Comment:
`QCoreApplication::applicationDirPath() ` can be removed. Just use `QDir::currentPath()`